packages/x-components/static-docs/experience-search-&-discovery/serp-ui.md

```diff
@@ -0,0 +1,109 @@
+---
+title: SERP UI
+---
+
+# SERP UI
+
+To handle and display search results, the layout of results and other related discovery features is
+paramount. For example, how to arrange results on the SERP or the features shoppers expect to easily
+navigate and organize results are key points to have in mind. <br/>
+
+![X Components for results layout](/assets/media/xcomponents_func_results_layout.svg)
+
+<FootNote>
+
+**X&nbsp;Components for results layout** <br/> (A) Grid, (B) Number of results, (C) Column Picker,
+(D) Sorting, and (E) Scrolling
+
+</FootNote>
+<br/>
+
+**Choice of the layout view** Your shoppers should be able to select how they’d like the products
+displayed and how the number of products they’d like to see per page, for example.
+
+That’s why the **grid** (A) is the core element of the SERP. It lists the results returned by the
```

Comment:
Actually, the ResultsList component, depicted in the `product-results-ui` is the responsible of list the result returned by the search. What the `Grid` does is to give layout to that list, in form of grid.

The grid can be used to display a grid of elements not related with the search, like recommendations

Reply:
What about sth like:
```suggestion
That’s why the **grid** (A) is the core element of the SERP. It displays the results returned by the  
```